polkadot/doc/testing.md

```diff
@@ -8,27 +8,59 @@ The testing strategy for Polkadot is 4-fold:
 
 ### Unit testing (1)
 
-Boring, small scale correctness tests of individual functions.
+Boring, small scale correctness tests of individual functions. It is usually
+enough to run `cargo test` in the crate you are testing.
+
+For full coverage you may have to pass some additional features. For example:
+
+```
+cargo test --features ci-only-tests
+```
 
 ### Integration tests
 
-There are two variants of integration tests:
+There are the following variants of integration tests:
 
 #### Subsystem tests (2)
 
 One particular subsystem (subsystem under test) interacts with a mocked overseer that is made to assert incoming and
-outgoing messages of the subsystem under test. This is largely present today, but has some fragmentation in the evolved
-integration test implementation. A `proc-macro`/`macro_rules` would allow for more consistent implementation and
-structure.
+outgoing messages of the subsystem under test. See e.g. the `statement-distribution` tests.
 
 #### Behavior tests (3)
 
 Launching small scale networks, with multiple adversarial nodes without any further tooling required. This should
 include tests around the thresholds in order to evaluate the error handling once certain assumed invariants fail.
 
-For this purpose based on `AllSubsystems` and `proc-macro` `AllSubsystemsGen`.
```

Comment:
I removed this stuff because I couldn't find any reference to `AllSubsystems` in the codebase.